src/patternfly/sass-utilities/scss-variables.scss

```diff
@@ -218,6 +218,7 @@ $pf-global--ListStyle: disc outside !default;
 
 // Transitions
 $pf-global--Transition: all 250ms cubic-bezier(.42, 0, .58, 1);
+$pf-global--toggle--Transition: .2s ease-in 0s;
```

Comment:
`transition-delay`'s initial value is `0s` so you shouldn't need that.

I'm also not sure about the var name. Changing a global var name is a fairly significant breaking change. Do we want to create transitions for behaviors, like spinning a toggle icon that are different from something like opening/closing a drawer or hiding the sidebar? FWIW, `.2s` is pretty similar to `250ms`, which is what we're already using for `$pf-global--TransitionDuration`. I also wonder if we should separate this out like the other var where we have maybe....

```
$pf-global--TransitionTimingFunction--100: ease-in;
$pf-global--TransitionTimingFunction--200: cubic-bezier(.645, .045, .355, 1);

$pf-global--TransitionDuration--100: .2s;
$pf-global--TransitionDuration--200: .25s;

$pf-global--Transition--toggle: $pf-global--TransitionDuration--100 $pf-global--TransitionTimingFunction--100;
$pf-global--Transition--menu: $pf-global--TransitionDuration--200 $pf-global--TransitionTimingFunction--200;
```

Although I think the vars we would make here depend on the different types of animations we want to offer, and I don't think we've explored that yet.

Maybe at a minimum I'd rename the var to `$pf-global--Transition--toggle;` as I imagine we'll have different variations of a transition, so the variation is a modifier of the thing it applies to. Or maybe we should wait and define the vars when we have a better transition/animation strategy. WDTY?

Reply:
@mcoker I really like the idea you have for separating out the vars etc. but I do think that we shouldn't make a global variable now when we don't know the full strategy behind the animation. I don't think theres any harm in keeping the values hard-coded and then creating a variable in the future?

Reply:
Agreed, sounds good!

Reply:
Agreed, sounds good!